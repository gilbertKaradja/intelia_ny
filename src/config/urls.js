let baseUrl = 'http://api.nytimes.com/svc/mostpopular';
let version = '/v2';

export const mostViewed = baseUrl + version + '/mostviewed';