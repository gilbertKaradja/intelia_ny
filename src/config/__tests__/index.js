import { mostViewed } from '../urls.js'

describe('URL configurations', () => {
    it('should match snapshots', () => {

        expect(mostViewed).toMatchSnapshot();

    });
})