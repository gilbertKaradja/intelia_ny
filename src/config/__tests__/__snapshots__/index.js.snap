// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`URL configurations should match snapshots 1`] = `"http://api.nytimes.com/svc/mostpopular/v2/mostviewed"`;
